CHANGES

```diff
@@ -2,6 +2,7 @@
 Features:
  - e-Reader card scanning
  - Add APNG recording
+ - Setting to display the file name of the currently loaded ROM instead of the game name in the title bar (closes mgba.io/i/1784)
```

Comment:
This isn't major enough to be marked as a feature (it's a visual tweak), so put this in Misc instead.